Terraform Registry publishing FAQ (#223)
hashicorp/terraform-docs-common: website/docs/registry/policy-libraries/publishing.mdx

We recommend publishing new versions to fix bugs instead of editing old versions of your policy library. When you edit the files in an old version of the library, users who have been depending on those policies cannot use them until they review your changes and update their Sentinel configuration files.

## Removing a Policy Library

> The owner of a policy library is able to remove it entirely from the Terraform Registry. Please be aware that [we recommend against removing anything from the Registry unless it contains a critical flaw](/terraform/registry/faq#how-do-i-remove-a-version-of-my-provider-module-or-policy-library-that-has-already-been-published).

Reviewer: ```suggestion
The owner of a policy library can remove that library from the Terraform Registry. Please be aware that [we do not recommend removing anything from the Registry unless it contains a critical flaw](/terraform/registry/faq#how-do-i-remove-a-version-of-my-provider-module-or-policy-library-that-has-already-been-published).
```
